src/utilities/formatters.js

```diff
@@ -74,6 +74,7 @@ export const formatDate = date => (date ? moment(date).fromNow() : generalString
  * @return  {string}              A string representing the date in 'yyyy-mm-dd' format
  */
 export const convertMobileDateToISO = dateString => {
+  dateString = moment(dateString).format('MM/DD/YYYY');
```

Comment:
https://github.com/openmsupply/mobile/issues/4640#issuecomment-1081250954

We could do something like this instead? Still brittle but just handles missing zeros better?
```
export const convertMobileDateToISO = dateString => {
  if (!dateString) return undefined;

  const separators = ['/', '-'];
  const splitDate = dateString.split(new RegExp(separators.join('|'), 'g'));

  if (splitDate.length !== 3) return undefined;
  const dayOfMonth = splitDate[0].padStart(2, '0');
  const month = splitDate[1].padStart(2, '0');
  const year = splitDate[2].padStart(4, '0');

  return `${year}-${month}-${dayOfMonth}`;
};
```

Reply:
Experimented a bit more, another alternative we could use if we'd still prefer to use moment (but the input format in both of these examples still needs to be DD/MM/YYYY and not MM/DD/YYYY):
`dateString = moment(dateString, 'DD/MM/YYYY').format('DD/MM/YYYY');`

Reply:
> 

sure I think we can do this... As we will just extract a day, month, and year whatever the format and whatever date it will be. I agree with your suggested solution.